### PR TITLE
[Blog] created a Blog for Introduction to Layer5 Learning Resources

### LINKED ISSUE #2323: [Blog] Publish the Learning path blogpost
Contributors and community members are encouraged to post on https://layer5.io/blog. A new blog post fitting the following description is requested to be written by a community member:

**Desired Title:**  Introducing Layer5 Learning Paths
**Draft:**  https://docs.google.com/document/d/1qqGluGEePjcBl2R79M_6h6wfu81xYaFcMgeSPLNDKt4/edit?usp=sharing
**Author:** Anita Ihuman

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by hirentimbadiya**
**Description**

This PR fixes #2323 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
👏 
@hirentimbadiya Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

**Comment by hirentimbadiya** [comment]
> 👏 @hirentimbadiya Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

okay 👍

File: src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+title: "Introduction to Layer5 Learning Resources"
+subtitle: Discover Learning Paths of Layer5
+date: 2023-07-22 05:23:00 -0530
+author: Hiren Timbadiya
```
Comment (vishalvivekm) [inline src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx:5]:
```suggestion
author: Layer5 Team
```

**Review by vishalvivekm — CHANGES_REQUESTED** [review]

File: src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx (hunk 1 above)
Comment (vishalvivekm) [reply src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx:5]: @hirentimbadiya

**Comment by l5io** [comment]
🚀 Preview for commit ca5d099dd1c9f0ffa5c2cf979694c92201c766f2 at: https://64be7e2b6fbae90506aafff8--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c9b3b566e50919a60cd5a7d641cdd1c25ad28ce4 at: https://64bfc2d4dd9b836459460039--layer5.netlify.app

File: src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    <li>Multiple Learning Paths.</li>
+    <li>User Account</li>
+    <li>Bookmarking / Progression tracking</li>
+    <li>Quizzes and grades</li>
+    <li>Certificate of Completion </li>
+  </ul>
+</div>
+
+<h2>Conclusion</h2>
+<p>Layer5 is a creator and maintainer of service mesh standards and this makes it the best place for you to gain this knowledge.  One of our fantastic initiatives is the interactive practice labs, which let you study while you explore services meshes on your browser. As the primary developers of the cloud native management plane, Meshery, our maintainers are readily available to answer your questions, should yin case you have any.
```
Comment (thisiskaransgit) [inline src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx:95]:
```suggestion
<p>Layer5 is a creator and maintainer of service mesh standards and this makes it the best place for you to gain this knowledge. One of our fantastic initiatives is the interactive practice labs, which let you study while you explore services meshes on your browser. As the primary developers of the cloud native management plane, Meshery, our maintainers are readily available to answer your questions, should in case you have any.
```

File: src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+<p>The Learning Path is an initiative developed to help enlighten and test your knowledge on service meshes and management of the workloads that you’ll build atop of them. As a light Learning Management System, Layer5 Learning Paths offer cloud-native training and certification through the lens of role-based paths with <Link to="/learn/service-mesh-labs"> interactive labs </Link> intertwined through the chapters. It comprises courses that are hand-curated by the maintainers at Layer5 to equip you with the foundational tools like <a href="https://meshery.io/">Meshery</a> that assists you in comprehending service mesh concepts.</p>
+
+<h2>Learning Path Options</h2>
+<p>Depending on your role as a developer, operator, or security engineer, you can choose the learning path most relevant to your everyday focus. This learning path currently has two focus areas for you to explore.</p>
+
+<ul>
+    <li><Link to="learn/learning-paths/mastering-service-meshes-for-developers">Mastering Service Meshes For Developers</Link> This introductory course in the Learning Path focuses on Developers and what they need to know to successfully design workloads for a service mesh environment.</li>
```
Comment (thisiskaransgit) [inline src/collections/blog/2023/2023-07-22-introduction-to-layer5-resources/index.mdx:33]: Correct the link here, please

File: .github/workflows/meshmap.yml
Comment (thisiskaransgit) [inline .github/workflows/meshmap.yml:1]: Revert this change, please

**Review by thisiskaransgit — COMMENTED** [review]
@hirentimbadiya, thank you for working on this, here are some points to consider
- Keep "service mesh"  lowercased for all instances, except for the hyperlinks
- Remove the hyphen between cloud native

**Comment by thisiskaransgit** [comment]
Also, we will be needing an updated thumbnail for the blog. // @ritiksaxena124

**Comment by l5io** [comment]
🚀 Preview for commit 37a15b8ea671b2e19c2708a797ef59fd4927d499 at: https://64c1e5af0ae95f3a19a225b6--layer5.netlify.app

**Comment by hirentimbadiya** [comment]
@thisiskaransgit, I have implemented some changes. Please review them, and  waiting for @ritiksaxena124 to upload the updated thumbnail image.

**Comment by l5io** [comment]
🚀 Preview for commit 485516818e4ef8d5df05a28ac0c901d626ec046f at: https://64c1eceae696c643b867de9c--layer5.netlify.app

**Comment by hirentimbadiya** [comment]
I have incorporated all the recommendations from @thisiskaransgit and have also updated the thumbnail provided by @ritiksaxena124. Please review this pull request, @vishalvivekm, @thisiskaransgit, @Chadha93.

**Comment by l5io** [comment]
🚀 Preview for commit 6666ffbc6186d92d6ed8748587dcd418de682f25 at: https://64c496a7eececd34ce7cdfc5--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 6666ffbc6186d92d6ed8748587dcd418de682f25 at: https://64c496a974848a367803a61d--layer5.netlify.app

**Comment by sudhanshutech** [comment]
Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Comment by l5io** [comment]
🚀 Preview for commit 9994f3ce2965ea76ca80ad8083336c6803c6da40 at: https://64c88ae1145ca43764a8f55a--layer5.netlify.app

**Comment by hirentimbadiya** [comment]
@vishalvivekm , @thisiskaransgit kindly review this PR.

**Comment by hirentimbadiya** [comment]
@ritiksaxena124 can you provide an image like this for blog ? 
learning-paths-architechture
